contracts/samples/DepositPaymaster.sol

```diff
@@ -131,7 +131,7 @@ contract DepositPaymaster is BasePaymaster {
 
         (userOpHash);
         // verificationGasLimit is dual-purposed, as gas limit for postOp. make sure it is high enough
-        require(userOp.verificationGasLimit > COST_OF_POST, "DepositPaymaster: gas too low for postOp");
+        require(uint128(bytes16(userOp.paymasterAndData[36:52])) > COST_OF_POST, "DepositPaymaster: gas too low for postOp");
```

Comment:
merge master, as we removed DepositPaymaster.

Reply:
Ok

Reply:
Ignore this file